mango_base/mango/dashboards/time_series_utils/file_uploader.py

```diff
@@ -0,0 +1,114 @@
+import pandas as pd
+import streamlit as st
```

Comment:
Review if this script .py must be here or in the time-series-dashboard branch.
@antoniogonzalezsuarez